### PR TITLE
Selectors for annotation formats

### PR DESCRIPTION
**Pull request by bsekachev**

### CODE REVIEW

File: cvat/apps/engine/templates/engine/annotation.html [hunk 1]
```diff
@@ -330,21 +330,21 @@
...
+            </select>
+
             <button id="removeAnnotationButton" class="menuButton semiBold h2"> Remove Annotation </button>
             <button id="settingsButton" class="menuButton semiBold h2"> Settings </button>
             <button id="fullScreenButton" class="menuButton semiBold h2"> Fullscreen Player </button>
             <button id="switchAAMButton" class="menuButton semiBold h2"> Switch AAM </button>
             <button id="helpButton" class="menuButton semiBold h2"> Help </button>
             <button id="saveButton" class="menuButton semiBold h2"> Save Work </button>
-            <input type="file" id="annotationFileSelector" style="display: none"/>
+            <input type="file" id="annotationFileSelector" accept="text/xml,application/zip,application/json" style="display: none"/>
```
Comment (nmanovic) [inline cvat/apps/engine/templates/engine/annotation.html]: Can we have an annotation format in `txt` format in the future? I like the idea to restrict the selection but really don't like that it is "hard coded".

File: cvat/apps/dashboard/static/dashboard/js/dashboard.js [hunk 2]
```diff
@@ -77,22 +77,21 @@ class TaskView {
...
-                button.text('Uploading..');
-                button.prop('disabled', true);
-                try {
-                    await this._task.annotations.upload(file, format);
-                } catch (error) {
-                    showMessage(error.message);
-                } finally {
-                    button.prop('disabled', false);
-                    button.text('Upload Annotation');
+        $('<input type="file" accept="text/xml,application/zip,application/json">')
```
Comment (nmanovic) [inline cvat/apps/dashboard/static/dashboard/js/dashboard.js]: Again the code need to fix in several places if add 'txt', 'yaml', ... formats. Previously we accepted only xml and it was OK.

File: cvat/apps/engine/templates/engine/annotation.html (hunk 1 above)
Comment (bsekachev) [reply cvat/apps/engine/templates/engine/annotation.html]: @nmanovic 
Browse window had been restricted before (at least in the dashboard). 
I just expanded it and did it consistent. Probably we can expand it dynamically.

Comment (bsekachev) [reply cvat/apps/engine/templates/engine/annotation.html]: Done

**Pull request merged**